railties/lib/rails/generators/rails/app/interactive_options.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+module Rails
+  module Generators
+    class InteractiveOptions # :nodoc:
+      include Database
+      include Thor::Shell
+
+      attr_reader :options
+
+      def self.perform(initial_options)
+        new(initial_options).perform
+      end
+
+      def initialize(initial_options)
+        @options = initial_options
+      end
+
+      def perform
+        say "Rails CLI #{Rails.version}", :green
+
+        case inquire_rails_version
+        when "dev"    then options[:dev] = true
+        when "edge"   then options[:edge] = true
+        when "master" then options[:master] = true
+        end
+
+        options[:database]             = inquire_database
+        options[:skip_action_cable]    = inquire_action_cable
+        options[:skip_action_mailer]   = inquire_action_mailer
+        options[:skip_action_mailbox]  = inquire_action_mailbox
+        options[:skip_active_storage]  = inquire_active_storage
+        options[:skip_action_text]     = inquire_action_text
+        options[:skip_bootsnap]        = inquire_bootsnap
+        options[:skip_jbuilder]        = inquire_jbuilder
+        options[:skip_javascript]      = inquire_javascript
+        options[:skip_spring]          = inquire_spring
+        options[:skip_sprockets]       = inquire_sprockets
+        options[:skip_system_tests]    = inquire_system_tests
+        options[:skip_turbolinks]      = inquire_turbolinks
+        options[:skip_webpack_install] = inquire_webpack
+        options[:webpack]              = inquire_webpack_library unless options[:skip_webpack_install]
```

Comment:
What do you think about reordering these prompts a bit?

I think it makes more sense to focus, say, on Rails stuff first, then JavaScript and Webpack.

Reply:
Sounds good 👍 I changed the order, let me know if that needs a little more re-ordering